src/components/MarkdownPage/MarkdownPage.js

```diff
@@ -19,8 +19,20 @@ import toCommaSeparatedList from 'utils/toCommaSeparatedList';
 import {sharedStyles} from 'theme';
 import createOgUrl from 'utils/createOgUrl';
 
+type Props = {
+  authors: Array<string>,
+  createLink: Function,
+  date?: string,
+  enableScrollSync?: boolean,
+  ogDescription: string,
+  location: Object,
+  markdownRemark: Object,
```

Comment:
Can we make this a better type? Like `Node` from the [shared `types`](https://github.com/reactjs/reactjs.org/blob/master/src/types.js)?

Reply:
I'm at a bit of a loss here, because all your comments relate to the same MarkdownPage component, and they seem to be a bit conflicting...Which of the changes should I do? :)

Reply:
I left comments under the lines that should be changed. (I think that's how GitHub's PR tool is typically used.)

<img width="521" alt="screen shot 2017-10-18 at 8 43 48 am" src="https://user-images.githubusercontent.com/29597/31728193-7c899636-b3e0-11e7-9d04-37a252a4484c.png">

So in this case, the comment is referring to the `markdownRemark` type. It's just a plain `Object` but it could be a more strict type- specifically one I've already created in [this file named `types`](https://github.com/reactjs/reactjs.org/blob/master/src/types.js).

Reply:
I just edited my previous comments to hopefully make what I'm asking for clearer. Please let me know if there's any confusion.